.env.example

```diff
@@ -1,12 +1,13 @@
 # 1. Copy this file and rename it to .env.local
 # 2. Update the enviroment variables below.
 
-# API key and secret. If you use LiveKit Cloud this can be generated via the cloud dashboard.
-LIVEKIT_API_KEY=devkey
-LIVEKIT_API_SECRET=secret
+# API key and secret. If you use LiveKit Cloud this can be generated via the cloud dashboard,
+# or by running `lk cloud auth` with the LiveKit CLI.
+LIVEKIT_API_KEY="$LIVEKIT_API_URL"
```

Comment:
Not sure I understand this change, this is an example .env file trying to help users to fill out the correct env vars.

Reply:
Probably makes a little more sense in context with [this PR](https://github.com/livekit/livekit-cli/pull/387) which implements the bootstrap commands.

The CLI will clone down a template repo, copy any `.env.example` to `.env.local`, replacing known placeholder values with real values. If your issue is with the magic string substitution, I'm sure we could come up with a solution that simply parses the env file for the presence of keys, not values.

Reply:
in my head it's mainly about `.env.example` being a template for users to set up their own env file. 
Using variable placeholders doesn't really help users in that regard and might rather be potentially confusing to them. 
We could of course add comments outlining the expected values, but if the approach I suggested in the other comment (using the bootstrap.yaml for env var mapping) seems reasonable to you, I'd (purely personally) prefer not mixing in things in the example files.